src/Resources/Resources/ChangeLog.md

```diff
@@ -22,6 +22,7 @@
 * Introduced secrets detection feature to safeguard sensitive data.
 * Migrated SDK generation from autorest csharp to autorest powershell.
 * Fixed deployment and deployment stack New/Set cmdlets to fail if template/parameter uri fails to downloads.
+* Added null check to the permissions object in the ToPSRoleDefinition method to return if the whole permissions object array is null.
```

Comment:
Please insert your change log between Line21-22 to avoid missing new changes.

Reply:
updated